docs/how_to_use_clusterctl.md

```diff
@@ -0,0 +1,52 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [IBM Cloud](#ibm-cloud)
+  - [How to use clusterctl image in existing kubernetes cluster?](#how-to-use-clusterctl-image-in-existing-kubernetes-cluster)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# IBM Cloud
+## How to use clusterctl image in existing kubernetes cluster?
+The `clusterctl` image is designed to run independently to provision ibmcloud cluster. we have embedded the `kind` and `kubectl` into `clusterctl` image.
+
+1. Follow [this readme](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/#cluster-creation) to generate `cluster.yaml`, `machines.yaml`, `provider-components.yaml`, and `addons.yaml`
+
+2. Create `serviceAccount` for clusterctl running
+    ```
+    kubectl apply -f examples/clusterctl/serviceaccount.yaml
+    ```
+3. Create `configmap` with `cluster.yaml`, `machines.yaml`, `provider-components.yaml`, and `addons.yaml`.
+    ```
+    kubectl create configmap ibmcloud-config --from-file=./cluster.yaml --from-file=./machines.yaml --from-file=./provider-components.yaml --from-file=./addons.yaml
```

Comment:
For some long command, you can update it as follows:

```
kubectl create configmap ibmcloud-config --from-file=./cluster.yaml \
    --from-file=./machines.yaml --from-file=./provider-components.yaml \
    --from-file=./addons.yaml
```

Ditto for other long command.